composer.json

```diff
@@ -27,7 +27,7 @@
 	"require": {
 		"php": ">=7.0",
 		"composer/installers": "1.*,>=1.0.1",
-		"mediawiki/semantic-media-wiki": "~3.0"
+		"mediawiki/semantic-media-wiki": "~3.0|~3.1"
```

Comment:
@mwjames I believe this was necessary?!

Reply:
According to [0] ~3.0 should be equivalent to >=3.0 <4.0.0

[0] https://getcomposer.org/doc/articles/versions.md

Reply:
You would need `|` when allowing different major versions as in case of `2.*|3.*`

Reply:
Thanks for the info!